docs/articles/instances.md

```diff
@@ -0,0 +1,26 @@
+# Instances
+
+## What is an Instance?
+Instances are the Marathon representation of [⇒Tasks](tasks.md) in Mesos. An Instance can exist even if there is currently no Task associated. Depending on the goal of a given Instance, Marathon will take actions to reach that goal: it might match offers in order to launch a Task or [⇒TaskGroup](taskGroups.md), or it might send kill requests to Mesos and eventually expunge the Instance from its state. See [⇒Instance Life Cycle](instanceLifeCycle.md) for more information.
```

Comment:
Wdyt about a small paragraph about `Goal`s, which ones exist, what are the allowed transitions etc.?

Reply:
Also, we should stay consistent with docs article naming e.g. (1) camel-case `instanceLifeCycle.md` vs (2) kebab-case e.g. `instance-life-cycle.md`. My vote goes to (2)

Reply:
👍 I adjusted the links to what I generated for index.md – which is (2) :)

I'll add a sentence or two wrt goals; I'd then link to _Instance Life Cycle_ as I think goals could be explained in more depth there. WDYT?